docs/apis/swagger-docs.yml

```diff
@@ -1029,4 +1033,235 @@ paths:
+                  uploadFileName: verification-jobs/845dae593a60280092554d2e125b3db7.mp3
+                  licenseStatus: Unverified
+                  songsConfirmedAt: null
+                  updatedAt: '2024-09-16T16:45:17.000Z'
+                  status: Complete
+                  songs:
+                    - title: Start a Fire
+                      artists: Who's Molly?
+                      endTime: 20
+                      startTime: 0
+                      cncMixId: null
+                      cncId: 5f0470bfb11d420440942a35
+                      composer: null
+                      isrc: GB89R1600173
+                      iswc: null
+                      publisher: null
+                      recordLabel: null
+                      tunecode: null
+                      yearOfRelease: null
+                      upc: '5054526103603'
+                      manualStatus: null
+                      licenseStatus: Unverified
+                      licenseChecks:
+                        Publishing Adaptation Rights: Unknown
+                        Publishing Duplication Rights: Unknown
+                        Publishing Choreography Rights: Unknown
+                        Title Matches: Unknown
+                        Artists Match: Unknown
+                        Full Ownership: Unknown
+                        Valid Date: Unknown
+                        License Source Matches: Unknown
+                        Licensee Name Match: Rejected
+                        Territory Match:
+                          US: Unknown
+                          GB: Unknown
+                      licenseData:
+                        bespokeAgreement: null
+                        licenseSource: Other
+                        cncLicense: null
+                        externalLicense:
+                          id: 133
+                          sourceName: Other Platform
+                          files:
+                            - id: 104
+                              downloadURL: >-
+                                http://lvs.local.clicknclear.com:3001/api/files/download?t=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJidWNrZXQiOiJjbGlja25jbGVhci12ZXJpZmljYXRpb24tbGljZW5zZXMtc3RhZ2luZyIsImtleSI6ImV4dGVybmFsLWxpY2Vuc2VzL2MxOTg2MjBlZGYyMmI5YzM3ZDRiNjliNjMyOWY1NDNlLnBkZiIsImZpbGVuYW1lIjoiVW5rbm93bi5wZGYiLCJjb250ZW50VHlwZSI6ImFwcGxpY2F0aW9uL3BkZiIsImlhdCI6MTcyNjU1ODMwNSwiZXhwIjoxNzI2NjQ0NzA1fQ.sJXEJbXqDSXXZheAxRKSts9WaywT3A-VlT5HlEO9aVY
```

Comment:
Just wanted to check if we wanted to include a local development server link example vs. say staging?

Reply:
As this is an example response body I think this ok.